code-challenges/filter/challenges-08.test.js

```diff
@@ -0,0 +1,320 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named oddValues that, given an array of integers as input, uses filter to return an array containing only the odd integers.
+
+For example, oddValues([1,2,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+
+const oddValues = (arr) => {
+  return arr.filter((a) => a % 2 === 1
+  );
+}
+
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named filterStringsWithVowels that, given an array of strings as input, uses filter to return an array with only words that contain vowels.
+
+The callback function to filter should include or utilize a regular expression pattern.
+
+For example, filterStringsWithVowels('gregor','hound','xyz') returns ['gregor', 'hound'].
+------------------------------------------------------------------------------------------------ */
+
+
+const filterStringsWithVowels = (arr) => {
+  // Solution code here...
```

Comment:
you have a lot of dead code here, make sure you remember to remove all the unnecessary code in the future